broker/nats/nats.go

```diff
@@ -64,6 +64,12 @@ func New(opts Options) (broker.Handler, error) {
 
 	return &Nats{ec: ec}, nil
 }
+func (n *Nats) ConnectedEndpoints() (endpoints []string) {
```

Comment:
Yes, from meshery server(client of broker)

Reply:
If by client, you mean as a graphql subscription or other client request, No. This is to be used internally by meshery server to keep track of address of connected brokers at any time. Although it can be exposed to client from some graphql subscription easily